src/main/docs/guide/configuration/sqs.adoc

```diff
@@ -14,6 +14,16 @@ and the following to your `build.gradle` or `pom.xml`
 
 dependency:io.micronaut.jms:micronaut-jms-sqs[]
 
+Force the `software.amazon.awssdk:sqs` dependency to `2.21.15`. For example, with gradle:
+
+[source,kotlin]
+----
+configurations.all {
+    resolutionStrategy {
+        force("software.amazon.awssdk:sqs:2.21.15")
+    }
+}
+----
 
```

Comment:
Will we need to update this every time we update aws or sqs?

How will we make sure we don't forget?

Reply:
I want to [do a major release of the module and update to `Jakarta.jms`](https://github.com/micronaut-projects/micronaut-jms/pull/474), and it should work without the need to pin to an old version. We should be able to remove this doc snippet and the forced pin.